File: ocmsdk/src/main/java/com/gigigo/orchextra/core/data/api/dto/content/ApiContentItem.java
package com.gigigo.orchextra.core.data.api.dto.content;

import com.gigigo.orchextra.core.data.api.dto.elements.ApiElement;
import java.util.List;

public class ApiContentItem {

  private String slug;
  private String type;
  private List<String> tags;
  private ApiContentItemLayout layout;
  private List<ApiElement> elements;

  public String getSlug() {
    return slug;
  }

  public String getType() {
    return type;
  }

  public List<String> getTags() {
    return tags;
  }

  public ApiContentItemLayout getLayout() {
    return layout;
  }

  public List<ApiElement> getElements() {
    return elements;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/data/api/dto/content/ApiContentItem.kt
package com.gigigo.orchextra.core.data.api.dto.content

import com.gigigo.orchextra.core.data.api.dto.elements.ApiElement

class ApiContentItem(
    val slug: String,
    val type: String?,
    val tags: List<String>?,
    val layout: ApiContentItemLayout?,
    val elements: List<ApiElement>?)